README.md

```diff
@@ -54,5 +56,11 @@ mkvirtualenv m05-mp-decaillet --python "C:\Users\%username%\AppData\Local\Progra
 ```
 
 _NB: exact path to **python3.11** may vary; locate it with: `where python` (Windows CMD) or `get-command python` (Windows PowerShell)_
-  
+
+<!-- -------------------------------------------------- -->
+
+## Unit tests
+* activate your virtualenv: `workon subtitles-creator`
+* run unit tests: `python -m unittest discover -v`
```

Comment:
We could add the "-s 'unit_tests" for the command to run for the whole directory

Reply:
no. it's a filter to run only certain dirs.
you're missing an __init__.py file